### DISCUSSION TITLE
Have not recieved Student Benefits yet such as Copilot.

### DISCUSSION THREAD

**Discussion by Devansh-Singhal** [community/community#130375] [GitHub Education]
### Hi everyone,

I have not yet received student benefits, including access to tools like GitHub Copilot. These benefits are essential for my projects. I need guidance on how to ensure I receive these benefits promptly.
![image](https://github.com/community/community/assets/72261386/155e71d4-af06-49e1-bf75-f707a2719f62)

**Comment by MohammedNadeemR** [comment]
response 2

**Reply by Devansh-Singhal** [reply]
what do you mean "response 2"?

**Comment by Jones-Sw** [comment]
What do you mean response 2

**Comment by irmo-de** [comment]
You will get another message when your account is fully setup. This takes up to 5 days.

**Comment by davevad93** [comment]
Hi @Devansh-Singhal ,

It usually takes a few days, but may take longer depending on different factors such as the volume of applications.

When the benefits are available, the status label will turn purple and display the following:

- `Congrats! Your request for @Devansh-Singhal was approved and your benefits are now available`

**Reply by JoeTeacherDAU** [reply]
Where are these messages found if I may ask?

**Reply by davevad93** [reply]
Hey @JoeTeacherDAU , you can check the status of the application [here](https://education.github.com/discount_requests/application).
